controllers/cruisecontroloperation_ttl_controller.go

```diff
@@ -0,0 +1,116 @@
+// Copyright © 2022 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controllers
+
+import (
+	"context"
+	"time"
+
+	"github.com/go-logr/logr"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+
+	"github.com/banzaicloud/koperator/api/v1alpha1"
+	banzaiv1alpha1 "github.com/banzaicloud/koperator/api/v1alpha1"
+)
+
+const (
+	defaultRequeueCruiseControlOperatonTTL = 30
```

Comment:
**Recommended**: Because this is an internal default value, I think we **SHOULD** be using a structured type for this to make it easier to use.

```suggestion
	defaultRequeueCruiseControlOperatonTTL = 30 * time.Seconds
```

Reply:
This can be remove because it is not used anywhere.